packages/vscode-lit-plugin/src/test/scripts/test-runner.ts

```diff
@@ -23,6 +23,10 @@ async function main() {
 		const fixturesDir = path.join(__dirname, "..", "..", "..", "src", "test", "fixtures");
 		// Download VS Code, unzip it and run the integration test
 		await runTests({ extensionDevelopmentPath, extensionTestsPath, launchArgs: [fixturesDir] });
+
+		setTimeout(function () {
```

Comment:
Is this a leftover?

Seems like it could cause a hanging test to exit with a success code and report a false success?